### DISCUSSION TITLE
[Feature]: AI assistant for building/editing?

### DISCUSSION THREAD

**Discussion by illtellyoulater** [GrapesJS/grapesjs#5558] [Roadmap Suggestions]
What about implementing an AI assistant mode for building new code or editing existing code? 
For example, checkout Pinegrow AI assistant: https://pinegrow.com/docs/ai-assistant/

**Comment by artf** [comment]
This could be easily integrated with a plugin, witnessed already many similar implementations with grapesjs.
Will try to find some time to do an example in GrapesJS Studio

**Reply by illtellyoulater** [reply]
I just found [**grapesjs-gpt-plugin**](https://github.com/sathudeva7/grapesjs-gpt-plugin)!

But why is it not listed in the [grapesjs plugins page](https://github.com/GrapesJS/grapesjs?tab=readme-ov-file#plugins)?

Are there any issues with it?

**Reply by artf** [reply]
Because it's necessary to add `grapesjs-plugin` as a topic 🥲

**Comment by ronaldohoch** [comment]
What chatGPT does in Pinegrow?

**Reply by illtellyoulater** [reply]
https://pinegrow.com/docs/ai-assistant/

**Comment by jcenlo** [comment]
i also was thinking about this, is only for the content? and the structure should be read from templates?
